defaults/main/install.yml

```diff
@@ -17,12 +17,19 @@ apache_conf_dir: "{{ apache_etc_dir }}/conf.d"
 apache_log_dir: "/var/log/{{ apache_service_name }}"
 
 rpm_repo_url: "https://yum.osc.edu/ondemand/2.0/ondemand-release-web-2.0-1.noarch.rpm"
+apt_repo_url: "https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest_2_all.deb"
```

Comment:
That deb release file will add wrong nodejs repo.  You want https://apt.osc.edu/ondemand/latest/ondemand-release-web_2.0.0_all.deb

Reply:
I changed this, but we `sed` to look at build right now anyhow, so I wonder if it matters until we get a 2.0 repo?

Reply:
I think so, that is getting the sed to point to the right repo. I think it's sorted now. Tests will show if that's the case!